docs/components/table.md

```diff
@@ -257,6 +254,113 @@ export default {
 </script>
 ```
 
+### Custom Row Attributes
```

Comment:
I think this should be moved out of the "Events" section and be its own before "slots". Also lets not make the border required to pass attrs. I would move `hasSideBorder` up above next to `clickable`,`small`, and `hover` and just say "by default each row has a left border. You can disable by passing false."

Then explain `rowAttrs` is a fn that allows you to bind any attribute to the table row. It passes the row details up so you can conditionally apply attributes. Then show the class example. 
"Here we demonstrate theming the `tableHover` variable based on row data., etc"

Reply:
question here: `border-spacing: 0 4px` and `border-collapse: separate` should we make this the default css then?

Reply:
not having any side borders would then turn the border-collapse and border-spacing back to the way it was